app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -1232,7 +1232,8 @@ class BrowserTabFragment :
 
     @SuppressLint("RequiresFeature")
     private fun configureDarkThemeSupport(webSettings: WebSettings) {
-        if (themingDataStore.theme == DuckDuckGoTheme.DARK) {
+        val currentNightMode = resources.configuration.uiMode and Configuration.UI_MODE_NIGHT_MASK
```

Comment:
should be fixed now @malmstein

Reply:
Actually, not yet. I will ping you once this is ready to review.

This is failing if you selected light theme but your device is in Dark mode.